package.json

```diff
@@ -29,7 +29,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@carbon/react": ">1.47.0 <1.50.0",
```

Comment:
Hi @elishabantana there's an ongoing discussion to make this a peer dependency all together. The reason it was previously like this was because of Carbon's searchable multi-select behavior between versions. 
We also wouldn't want to give a static version because this is package is only a library and not a microfrontend

Reply:
@pirupius sorry I'm a bit lost. Does that mean we shouldn't update carbon or we should update it to be a more up-to-date range?

Reply:
@NethmiRodrigo The reason this was added was to resolve a breaking change in some components. The solution would involve
1. Testing and fixing any breaking changes in some components with newer versions Carbon 
2. Move carbon back to being a peer dependency.